src/include/network/lan/help.rb

```diff
@@ -98,8 +98,12 @@ def initialize_network_lan_help(_include_target)
                                ) +
                                  _(
                                    "<p>For each route, enter destination network IP address, gateway address,\n" \
-                                     "and netmask. To omit any of these values, use a dash sign \"-\". Select\n" \
-                                     "the device through which the traffic to the defined network will be routed.\"-\" is an alias for any interface.</p>\n"
+                                     "and netmask. You can use either IPv4 netmask or prefix lenght when defining\n" \
```

Comment:
If that is the only grammar issue, I have to congratulate myself ;-) 

Thanks, fixing.